class DataSourceException implements Exception {
  const DataSourceException({this.message});

  final String? message;
}

//Messages Exception
const dataSourceException = 'Erro ao executar essa operação';
